_providers/wkpb.de.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Westfalen-Kolleg Paderborn
+status: OK
```

Comment:
even if technically `OK` is fine, this will show the hint already now on all platforms (apart from that, there are currently no other side effects)

```suggestion
status: PREPARATION
```